src/Widgets/AbstractAppContainer.vala

```diff
@@ -73,7 +73,18 @@ namespace AppCenter {
                     return false;
                 }
 
-                return this.package.get_payments_key() != null;
+                return this.package.get_payments_key () != null;
+            }
+        }
+
```

Comment:
Since the `suggested_amount` is cached within the Package itself, there is no need to have another private one that's basically doing the same thing.